.github/workflows/build_conda_pkg.yml

```diff
@@ -35,6 +35,9 @@ jobs:
           source test_python/bin/activate
           make installdeps-test
       - name: Clone Feedstock, Copy to Container, and Run Update
+        env:
+          DOCKERHUB_USER: ${{ secrets.DOCKERHUB_USER }}
+          DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
I don't think this is necessary per se, but I think it helps disambiguate the fact that the double-curlies are for the GH secret, and then the "${}" below are for the shell scripting 

https://docs.github.com/en/actions/reference/encrypted-secrets#example-using-bash